CHANGELOG.md

```diff
@@ -1,34 +1,71 @@
 # Change Log
-This file documents all notable changes to juttle. The release numbering uses [semantic versioning](http://semver.org).
 
+This file documents all notable changes to Juttle. The release numbering uses [semantic versioning](http://semver.org).
+
+## 0.3.0
+
+Released 2016-01-20
+
+### Major Changes
+
+- Added support for points with array/object field values. [[#55](https://github.com/juttle/juttle/issues/55)]
+- Added support for Grok parsing to `file` and `stdio` adapters (`-format "grok" -pattern ...`). [[#37](https://github.com/juttle/juttle/issues/37)]
+- Adapters are now loaded only when they are used. [[#83](https://github.com/juttle/juttle/issues/83)]
```

Comment:
This also changed the way adapters are configured -- we should mention that as well.